selene-lib/tests/lints/standard_library/assert.lua

```diff
@@ -0,0 +1,4 @@
+assert(true, 'message')
```

Comment:
Can you replace this and the other one with double quotes just to be consistent?